libs/ui-toolkit/icon/src/lib/icon.component.ts

```diff
@@ -22,24 +25,34 @@ export class IconComponent {
    * The icon modifier for the icon path.
    */
   @Input() public set icon(data: string) {
+    this.imageError = false;
     this._icon = data;
     this.updatePath();
   }
 
   /**
-   * Base prefix for the icon path.
+   * Secondary option for icon base URL/path.
+   */
+  @Input() public set secondaryBase(data: string) {

```

Comment:
I think that secondary base path shouldn't be a input, the fallback must be always for `/` to search by image in project